### ISSUE TITLE
EC2 instances generated by AGC don't have descriptive tags

### ISSUE THREAD

**Issue by alexiswl** [aws/amazon-genomics-cli#49]
Hello, I've noticed that the tags (and name) of ec2 resources provisioned by AGC don't have descriptive tags, making it difficult to keep track of resources that have been spun up from AGC.

As an example:

```
$ aws ec2 describe-tags --filters "Name=resource-id,Values=i-067cdd56997ed1d0f"
{
    "Tags": [
        {
            "Key": "aws:autoscaling:groupName",
            "ResourceId": "i-067cdd56997ed1d0f",
            "ResourceType": "instance",
            "Value": "TaskBatchComputeEnviron-b509e63dc5ffd5b-asg-32d3f092-50e7-332e-84a4-f28e5a3f894a"
        },
        {
            "Key": "aws:ec2launchtemplate:id",
            "ResourceId": "i-067cdd56997ed1d0f",
            "ResourceType": "instance",
            "Value": "lt-05ebe391321e46034"
        },
        {
            "Key": "aws:ec2launchtemplate:version",
            "ResourceId": "i-067cdd56997ed1d0f",
            "ResourceType": "instance",
            "Value": "1"
        }
    ]
}
```

We ideally would like to see a `name` and `stack` tag for our agc instances.  

```
$ aws ec2 describe-instances --instance-ids i-067cdd56997ed1d0f
{
    "Reservations": [
        {
            "Groups": [],
            "Instances": [
                {
                    "AmiLaunchIndex": 0,
                    "ImageId": "ami-0310d0e01c1e033c0",
                    "InstanceId": "i-067cdd56997ed1d0f",
                    "InstanceType": "m5.large",
                    "LaunchTime": "2021-09-29T23:30:37+00:00",
                    "Monitoring": {
                        "State": "disabled"
                    },
                    "Placement": {
                        "AvailabilityZone": "ap-southeast-2b",
                        "GroupName": "",
                        "Tenancy": "default"
                    },
                    "PrivateDnsName": "",
                    "ProductCodes": [],
                    "PublicDnsName": "",
                    "State": {
                        "Code": 48,
                        "Name": "terminated"
                    },
                    "StateTransitionReason": "User initiated (2021-09-29 23:34:15 GMT)",
                    "Architecture": "x86_64",
                    "BlockDeviceMappings": [],
                    "ClientToken": "b605f0cf-950d-acc1-6993-44386e49162c",
                    "EbsOptimized": false,
                    "EnaSupport": true,
                    "Hypervisor": "xen",
                    "NetworkInterfaces": [],
                    "RootDeviceName": "/dev/xvda",
                    "RootDeviceType": "ebs",
                    "SecurityGroups": [],
                    "StateReason": {
                        "Code": "Client.UserInitiatedShutdown",
                        "Message": "Client.UserInitiatedShutdown: User initiated shutdown"
                    },
                    "Tags": [
                        {
                            "Key": "aws:ec2launchtemplate:id",
                            "Value": "lt-05ebe391321e46034"
                        },
                        {
                            "Key": "aws:autoscaling:groupName",
                            "Value": "TaskBatchComputeEnviron-b509e63dc5ffd5b-asg-32d3f092-50e7-332e-84a4-f28e5a3f894a"
                        },
                        {
                            "Key": "aws:ec2launchtemplate:version",
                            "Value": "1"
                        }
                    ],
                    "VirtualizationType": "hvm",
                    "CpuOptions": {
                        "CoreCount": 1,
                        "ThreadsPerCore": 2
                    },
                    "CapacityReservationSpecification": {
                        "CapacityReservationPreference": "open"
                    },
                    "HibernationOptions": {
                        "Configured": false
                    },
                    "MetadataOptions": {
                        "State": "pending",
                        "HttpTokens": "optional",
                        "HttpPutResponseHopLimit": 1,
                        "HttpEndpoint": "enabled"
                    },
                    "EnclaveOptions": {
                        "Enabled": false
                    }
                }
            ],
            "OwnerId": "843407916570",
            "RequesterId": "081202882002",
            "ReservationId": "r-019f44509c65aeb4f"
        }
    ]
}
```

**Comment by wleepang** [comment]
This should be address in PR #38 and incorporated in the next build

**Comment by wleepang** [comment]
Fixed in release v1.0.1
